src/order/basic.lean

```diff
@@ -461,6 +598,14 @@ protected theorem monotone.order_dual {f : α → γ} (hf : monotone f) :
   @monotone (order_dual α) (order_dual γ) _ _ f :=
 λ x y hxy, hf hxy
 
+protected lemma monotone.order_dual_left {f : α → γ} (hf : monotone f) :
```

Comment:
For `strict_mono_on`, you used the terminology `dual`, here you are using `order_dual`. I have a mild preference for the former (what else than `order` could it be?), but in any case coherence would be good!